server/routes/probationPractitionerReferrals/probationPractitionerReferralsController.ts

```diff
@@ -208,7 +208,6 @@ export default class ProbationPractitionerReferralsController {
   async showReferral(req: Request, res: Response): Promise<void> {
     const { accessToken } = res.locals.user.token
     const sentReferral = await this.interventionsService.getSentReferral(accessToken, req.params.id)
-    const showSuccess = !!req.query.success
```

Comment:
would it not be a bit easier to read to keep this `showSuccess` as
```
    const showSuccess = req.query.detailsUpdated === 'true'
```
just shows the reason for this better than determining the boolean as its being passed in to `ShowReferralPresenter`

Reply:
fair point. i figured readers of this method don't care why we're pulling that query param out though, since there is no logic that depends on it here. happy to move it into a variable though.

Reply:
yeah, just a nitpick really, so doesn't really matter, happy for it to stay as is